lib/refer/controller.rb

```diff
@@ -12,14 +12,12 @@ def refer(referee, cookie_name: Refer.cookie_name)
       Refer.refer(code: cookies[cookie_name], referee: referee)
     end
 
-    private
```

Comment:
What's the use case for making this public? I don't see why you'd want to call this externally, it'll always be inside a controller.

Reply:
I thought that private methods inside controller concerns aren't available outside the controller concern. But I now learned that this is not true 😄

So yes, it should be private.